packages/@bbc/old-optimo-slate-react/package.json

```diff
@@ -1,7 +1,10 @@
 {
-  "name": "slate-react",
+  "name": "@bbc/old-optimo-slate-react",
   "description": "A set of React components for building completely customizable rich-text editors.",
-  "version": "0.21.0",
+  "version": "0.21.2",
```

Comment:
0.21.1?

Reply:
(well we thought we did at the time, it did actually work!)

Reply:
Ah - need to update the other PR to use 0.21.2 then?

Reply:
I think the two versions should be the same